x-pack/plugins/observability/public/pages/rules/index.tsx

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect, useCallback } from 'react';
+import moment from 'moment';
+import {
+  EuiBasicTable,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+  EuiText,
+  EuiBadge,
+  EuiPopover,
+  EuiContextMenuPanel,
+  EuiContextMenuItem,
+  EuiHorizontalRule,
+  EuiAutoRefreshButton,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { usePluginContext } from '../../hooks/use_plugin_context';
+import { useBreadcrumbs } from '../../hooks/use_breadcrumbs';
+
+import { useKibana } from '../../utils/kibana_react';
+
+import { loadRules } from '../../../../triggers_actions_ui/public';
+
+const DEFAULT_SEARCH_PAGE_SIZE: number = 25;
+
+interface RuleState {
+  data: [];
+  totalItemsCount: number;
+}
+
+interface Pagination {
+  index: number;
+  size: number;
+}
+
+export function RulesPage() {
+  const { core, ObservabilityPageTemplate } = usePluginContext();
+  const { docLinks } = useKibana().services;
+  const {
+    http,
+    notifications: { toasts },
+  } = core;
+  const [rules, setRules] = useState<RuleState>({ data: [], totalItemsCount: 0 });
+  const [page, setPage] = useState<Pagination>({ index: 0, size: DEFAULT_SEARCH_PAGE_SIZE });
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  async function loadObservabilityRules() {
+    try {
+      const response = await loadRules({
+        http,
+        page: { index: 0, size: DEFAULT_SEARCH_PAGE_SIZE },
+        typesFilter: [
+          'xpack.uptime.alerts.monitorStatus',
```

Comment:
Q: there is no module/constant file that provides these static values e.g. O11y rule types?

Reply:
True no such file at the moment. I don't like this either, I will address very soon in this issue https://github.com/elastic/kibana/issues/126775